drivers/virtio/Kconfig

```diff
@@ -0,0 +1,50 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig DRIVERS_VIRTIO
+	bool "Virtio Device Support"
+	depends on !OPENAMP
+	default n
+	---help---
+
+menuconfig DRIVERS_VIRTIO_NUM
```

Comment:
should we use more specific name(e.g. DRIVERS_VIRTIO_MMIO_NUM)? since all similar setting is specific for virtio-mmio transport(there are many other transport).

Reply:
done